correction_odom:
  ros__parameters:
    use_sim_time: False

    correction_service: /correction_odom
    robot_data_topic: /web_transform_node/robot_data
    set_odom_service: /chassis/reset_odom
    waypoints_topic: /waypoints

pub_waypoints:
  ros__parameters:
    use_sim_time: False
    waypoints_path: 国赛路径点b.yaml
    waypoints_topic: /waypoints

generate_path:
  ros__parameters:
    use_sim_time: False
    waypoints_topic: /waypoints
    generate_path_service: /generate_path

navigation_to_waypoint:
  ros__parameters:
    use_sim_time: False

    robot_data_topic: /web_transform_node/robot_data
    set_odom_service: /chassis/reset_odom
    navigation_to_waypoint_action: /navigation_to_waypoint
    generate_path_service: /generate_path
    correction_service: /correction_odom
    waypoints_topic: /waypoints
    navigation_action: /nav2_ptp_action

    speed: 0.4
    uphill_offset: 0.17
    reverse_uphill: True
    distance_threshold: 0.05
